utils/datautils.js

```diff
@@ -48,6 +49,39 @@ function getClaimById(claimId) {
   return db.Claim.findById(claimId)
 }
 
+function getConflictedClaims(claim,issueUrlDetail) {
+  projectName = '/' + issueUrlDetail.project + '/'
+  issueId = ['/' + issueUrlDetail.id + '/', '/' + issueUrlDetail.id ]
+  return db.Claim.findAll({
+    where : {
+      [Op.and] : [
+        {
+          issueUrl: {
+            [Op.like]: '%' + projectName + '%'
+          }
+        },
+        {
+          [Op.or] : [
+            {
+              issueUrl: {
+                [Op.like]: '%' + issueId[0]
```

Comment:
It won't work with trailing slash because you are not allowing any char to be after the id,

Reply:
> @prathambatra remove the semantic changes from the diff.

I haven't done those changes. How did they come? there is also an issue regarding that